homeassistant/components/roborock/const.py

```diff
@@ -49,3 +49,5 @@
 MAP_SLEEP = 3
 
 GET_MAPS_SERVICE_NAME = "get_maps"
+SET_VACUUM_GOTO_POSITION_SERVICE_NAME = "set_vacuum_goto_position"
```

Comment:
Would `get_vacuum_position` and `set_vacuum_positiion` work?

I'm not sure that `goto` and `current` add much. I get the idea that its setting a destination and a current position, but i think that detail may not matter unless we plan to expose `get_vacuum_goto_position`

Reply:
if you want it done differently, I have no problem changing it. The idea behind set_vacuum_goto_position is that you're not actually setting the robot’s position but rather the position it should drive to. As for get_vacuum_current_position, the thought is that there might be several positions you can query, such as the go-to position as well.

Reply:
With that logic i would think `target` would work better than `goto`

Reply:
I doubt there will be more action calls for positions than these, but I anyway think we should keep them to be specific for what they are (they are not some generic vacuum calls).

Reply:
Good point. The name “goto” basically comes from the API, and I’d also be in favor of “target.” In my German app, the function is called “Pin-and-Go.” How is it in the English version—does it also use “Pin-and-Go,” or something with “goto” in it?

Reply:
> With that logic i would think `target` would work better than `goto`

For vacuums , it's typically called "Go to xxx" in the apps, on the other hand for roborock they seem to have changed it into "Pin n go" (user facing).

Reply:
May I suggest a follow-up PR then to have the service names to be
- `set_vacuum_target_position` (from suggestion above)
- `get_vacuum_position` (the position would always be current I guess)

Let's replace the name of the set service to `Pin n go` (same as app).

Reply:
I mean we can put the technical name to what it does (because this just sounds awkward) and have the user facing name be the one is used in the app

Reply:
But my reasoning is mostly that it reads awkward, i dont have that strong of an opinion

Reply:
Same with me, all good suggestions but I have no preference

Reply:
@RaHehl feel free to open up a PR for renaming and we can agree there for the final name but I guess we're agreeing with the up. Let's get @allenporter to chime in later when he's online.

Reply:
I wanted to do a test later for a "clean room 1,3,4" service, maybe we can just add that in there, as the appropriate name is still pending there too

Reply:
I think that should not be part of that PR anyhow (which also doesn't exist yet) so I suggest to open one up anyhow.
A `clean segment` PR would not be related to the naming of these.

Reply:
As announced, there is clean segment pr here, I would be happy to receive your feedback https://github.com/home-assistant/core/pull/134148

Reply:
`set_vacuum_target_position` and `get_vacuum_position` seem fine to me, thanks.